### DISCUSSION TITLE
Payouts

### DISCUSSION THREAD

**Discussion by euluizaraujo** [juspay/hyperswitch#2953] [Q&A] [answered]
Hello, I would like to better understand this Payouts function, I noticed that adyen is available in both the standard connectors section and in payouts processors, what is the difference?

Is this payouts functionality available open source?

**Comment by kashif-m** [comment]
Hey,

Yes, the payouts feature is open source.

When it comes to Adyen, it handles both regular payments and payouts. To set up payouts through Adyen, you'll use something called a Payout Processor. It's different from the Payment Processors you'd use for regular payments. The hyperswitch repository has a clear distinction between the functionalities offered by these processors, and that's why you see separate listings for payment and payout processors.

If you have any more questions or need further clarification, feel free to ask!

**Reply by euluizaraujo** [reply]
Hello, thank you very much for your feedback. What would be the difference between regular payment and payments? Do you have any material to understand the difference in practice? @kashif-m

**Reply by kashif-m** [reply, marked as answer]
Both payments and payouts are basically movement of funds from one account to another, so nothing changes there.

But from an entity's POV, transactions which involves money moving in to your account would be a payment. And transactions involving money moving out of your account would be a payout.

I do not have any relevant material for understanding the differences. There's this article which will help you in understanding payouts better - https://dev.to/kshf/payouts-101-3g4m

**Reply by euluizaraujo** [reply]
@kashif-m thanks! Is the payout operation only via API or is there some front for this?

**Reply by kashif-m** [reply]
For now, payout operations are provided only via APIs. It will be available as a part of HyperSwitch dashboard soon. This will let you create and control your payouts seamlessly.

You can try out HyperSwitch's dashboard here - https://app.hyperswitch.io

**Reply by euluizaraujo** [reply]
@kashif-m Perfect. The payout API is already 100% open source, isn't it, is there a plan?

**Reply by kashif-m** [reply]
Yes, HyperSwitch is 100% open source and free to use. Although there is a HyperSwitch hosted version which is chargeable based on your API usage. You can reach out to hyperswitch@juspay.in for more details on using the hosted enterprise version.

Also, some of the payout use cases available at the moment are listed below -

- Processing one time payouts — plain, simple one-time payout transactions
- Processing payouts to customers with saved payment methods at HyperSwitch — any saved payment methods for a given customer can be directly used as a payout method for that customer.
- [UnderDevelopment] Recurring payouts — configure cron rule to process payout transactions recursively
- [UnderDevelopment] Processing bulk payouts — make bulk payout transactions using CSV / JSON
- [UnderDevelopment] Scheduled payouts — schedule one time / recurring future payouts

Some of the connectors onboarded in HyperSwitch -

- Adyen
- Stripe Connect
- Wise

Feel free to reach out if you have any questions!

**Reply by euluizaraujo** [reply]
@kashif-m Perfect, if you want to make the figma file of the payout management functionality available in the control center, I can help implement it.

**Reply by kashif-m** [reply]
Sure, that would be helpful. Will get back to you w the flows.
